tests/functional/watcher_controller_test.go

```diff
@@ -887,7 +885,7 @@ var _ = Describe("Watcher controller", func() {
 			Expect(WatcherAPI.Spec.ServiceAccount).To(Equal("watcher-watcher"))
 			Expect(int(*WatcherAPI.Spec.Replicas)).To(Equal(2))
 			Expect(*WatcherAPI.Spec.NodeSelector).To(Equal(map[string]string{"foo": "bar"}))
-			Expect(WatcherAPI.Spec.TLS.CaBundleSecretName).Should(Equal("combined-ca-bundle"))
+			Expect(WatcherAPI.Spec.TLS.Ca.CaBundleSecretName).Should(Equal("combined-ca-bundle"))
```

Comment:
The TLS under watcherapi.Spec has not changed in this patch, do we really need this change?

Reply:
yes, good catch we don't need this one, I was grepping for the occurrences in the specs of the other services and this one slipped in, I've changed it , thanks!